src/app/app.component/app.component.html

```diff
@@ -0,0 +1,17 @@
+<body>
+  <mat-drawer-container class="example-container">
+    <mat-drawer mode="side" opened>
+      <a class="navbar-brand" routerLink="/">ManZaf</a>
+      <div>
+        <h3>Single line lists</h3>
+        <mat-list>
+          <mat-list-item routerLink="/own-leave" style="cursor:pointer">My leaves</mat-list-item>
```

Comment:
lepiej unikac inline style, zrob sobie klase css i ja uzywaj, https://www.thoughtco.com/avoid-inline-styles-for-css-3466846.